### PR TITLE
UIPFUDP-39: remove unused

### PR DESCRIPTION
**Pull request by elsenhans**
https://folio-org.atlassian.net/browse/UIPFUDP-39

### CODE REVIEW

**Comment by alb3rtino** [comment]
There's quite a bit of code formatting going on in this PR, which makes it hard to review the functional changes. While the formatting updates are valid, would it be possible to move them to a separate PR? That would make reviewing much easier. Thanks!

**Review by alb3rtino — APPROVED** [review]

File: src/UDPsView.test.js [hunk 1]
```diff
@@ -37,16 +31,13 @@ const renderUDPsView = (
             queryGetter={jest.fn()}
             querySetter={jest.fn()}
             searchString=""
-            visibleColumns={['label', 'harvestingStatus', 'latestStats', 'aggregator']}
```
[the remark this answers is not in the document]
Comment (elsenhans) [reply src/UDPsView.test.js:40]: Removed `isEmptyMessage` and `onSearchComplete`

File: src/UDPsView.test.js [hunk 2]
```diff
@@ -110,12 +101,13 @@ describe('UDPsView', () => {
 
   describe('click "active" filter and load new results', () => {
     it('should show a certain amount of results and all columns', async () => {
```
[the remark this answers is not in the document]
Comment (elsenhans) [reply src/UDPsView.test.js]: Adapted test description.

File: src/UDPsView.test.js [hunk 3]
```diff
@@ -126,7 +118,7 @@ describe('UDPsView', () => {
 
   describe('enter search string and load new results', () => {
     it('should enable search button and show a certain amount of results', async () => {
```
[the remark this answers is not in the document]
Comment (elsenhans) [reply src/UDPsView.test.js]: Adapted test description and test.

**Pull request merged**